charts/gitea/templates/gitea/statefulset.yaml

```diff
@@ -103,6 +103,10 @@ spec:
             {{- end }}
           {{- if .Values.gitea.livenessProbe.enabled }}
           livenessProbe:
+            {{- toYaml .Values.gitea.customLivenessProbe | nindent 12 }}
+          {{- end }}
+          {{- if .Values.gitea.readinessProbe.enabled }}
+          readinessProbe:
```

Comment:
@Morriz not sure why this has gotten in here, what should be done about this? Keep, or revert?